tracer/test/Directory.Build.props

```diff
@@ -12,20 +12,40 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="6.4.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
     <PackageReference Include="Moq" Version="4.16.0" />
-    <PackageReference Include="xunit" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
+    <PackageReference Include="xunit" Version="2.4.2" />
     <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
   </ItemGroup>
 
+  <!-- Have to use the latest version of the test SDK on .NET Framework so that code coverage works -->
+  <!-- But this version does not support .NET Core 2.1/3.0, so need to ifdef here -->
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5" />
+    <!-- Note that we're using a custom version of coverlet from the Azure feed that supports StrongNaming -->
+    <PackageReference Include="coverlet.collector" Version="3.2.1-preview.4.g46dda4ea49">
```

Comment:
Is there documentation on how to create this NuGet? I didn't realize that this was something that we created

Reply:
Neither did I 😅 It's a fork of Coverlet [here](https://github.com/DataDog/coverlet) - there are branches that support StrongNaming in there. Then you just build locally and push the NuGet to Azure Devops (we have documentation for that part in Confluence)

Reply:
Thanks!